config.go

```diff
@@ -86,6 +86,9 @@ type Config struct {
 
 	// DNS config.
 	DNS config.DNS `json:"dns"`
+
+	// S3 Bucket Name
+	S3BucketName string
```

Comment:
Wonder if we should have provider specific config for stuff like this, since the bucket is only used for deploys to Lambda/AWS. Or we could maybe put it in `.lambda`'s object.

Also needs a `json` tag.